docker-for-windows/install-windows-home.md

```diff
@@ -0,0 +1,146 @@
+---
+description: How to install Docker Desktop on Windows 10 Home
+keywords: Windows 10 Home, Home, Windows, install, download, run, Docker, local
+title: Install Docker Desktop on Windows Home
+---
+
+You can now install Docker Desktop on Windows 10 Home machines. Docker Desktop for WSL 2 Windows Home is a full version of Docker Desktop for Linux container development.
+
+This page contains information on installing Docker Desktop on Windows 10 Home. If you are looking for information about installing Docker Desktop on Windows 10 Pro, Enterprise, or Education, see [Install Docker Desktop on Windows](/install.md).
+
+[Download from Docker Hub](Add URL){:
+.button .outline-btn}
+
+Docker Desktop for WSL 2 Windows Home offers the following benefits:
```

Comment:
"Docker Desktop for WSL 2 Windows Home": is it a thing ?

Reply:
Would it be better to say `Docker Desktop on Windows Home with the WSL 2 backend offers...`? Any other suggestions welcome :)

Reply:
Updated as per our discussion in Slack.